eppoclient/utils_test.go

```diff
@@ -26,7 +37,8 @@ func TestToFloat64(t *testing.T) {
 				t.Errorf("ToFloat64(%v) error = %v, expectErr %v", tt.input, err, tt.expectErr)
 				return
 			}
-			if !tt.expectErr && result != tt.expected {
+			closeEnough := math.Abs(result - tt.expected) < 0.00001
```

Comment:
"If I had a dime for every time I've seen someone use FLOAT to store currency, I'd have $999.997634" -- Bill Karwin.

Reply:
minor: this is problematic as this comparison value is arbitrary. Float precision depends on the number's exponent, and `0.00001` may be too high for some values (it could easily be higher than `result` itself).

I suggest that we use absolute relative error (`|(result - expected)/result|`) or design test cases that we don't need `closeEnough` instead.

Reply:
Will normalize--good call!

Reply:
Found that there's an `assert.InEpsilon()` function for this.